doc/source/examples/multiphysics/rising-bubble/rising-bubble.rst

```diff
@@ -95,57 +95,15 @@ The ``source term`` subsection defines the gravitational acceleration:
-The following equations calculate the filtered phase fraction gradient and filtered curvature, respectively.
-
-.. math:: 
-
-    \int_\Omega \left( {\bf{v}} \cdot {\bf{\psi}} + \eta_n \nabla {\bf{v}} \cdot \nabla {\bf{\psi}} \right) d\Omega = \int_\Omega \left( {\bf{v}} \cdot \nabla {\phi} \right) d\Omega
-
-where :math:`{\bf{v}}` is a vector test function, :math:`\bf{\psi}` is the filtered phase fraction gradient, :math:`\eta_n = \alpha h^2` is the phase fraction gradient filter value with :math:`h` denoting the cell size, and :math:`\phi` is the phase fraction.
-
-.. math::
-
-    \int_\Omega \left( v k + \eta_k \nabla v \cdot \nabla k \right) d\Omega = \int_\Omega \left( \nabla v \cdot \frac{\bf{\psi}}{|\bf{\psi}|} \right) d\Omega
-
-where :math:`k` is the filtered curvature, and :math:`\eta_k = \beta h^2` is the curvature filter value, and :math:`v` is a test function.
-
-.. tip::
-
-  The phase fraction gradient filter value (:math:`\eta_n = \alpha h^2`) and curvature filter value (:math:`\eta_k = \beta h^2`) must be small values larger than 0. The values of :math:`\alpha` and :math:`\beta` are controlled respectively by the parameters ``phase fraction gradient filter factor`` and ``curvature filter factor``  in the parameter file.
-  We recommend the following procedure to choose a proper value for these parameters:
+In the ``VOF`` subsection, three features are enabled : the ``interface sharpening``, the ``phase filtration`` and the ``surface tension force``.
 
-  1. Enable ``output auxiliary fields`` to write filtered phase fraction gradient and filtered curvature fields.
-  2. Choose a value close to 1, for example, the default values  :math:`\alpha = 4` and :math:`\beta = 1`.
-  3. Run the simulation and check whether the filtered phase fraction gradient and filtered curvature fields are smooth and without oscillation.
-  4. If the filtered phase fraction gradient and filtered curvature fields show oscillations, increase the value :math:`\alpha` and :math:`\beta` to larger values, and repeat this process until reaching smooth filtered phase fraction gradient and filtered curvature fields without oscillations. Generally, the default values should be sufficient.
+The interface sharpening method and its parameters are explained in the :doc:`../dam-break/dam-break` example. The phase filtration filters the phase field used for the calculation of physical properties by stiffening the value of the phase fraction. We refer the reader to the :doc:`../../../../parameters/cfd/volume_of_fluid` documentation for more explanation on the phase filtration. Finally, the surface tension force computation is explained in the :doc:`../static-bubble/static-bubble` example.
```

Comment:
Maybe it would be better to migrate the formulas into the theory guide for the VOF equations instead of distributing them like this across examples.

Reply:
Yes I agree, it is in my to-do list :)